src/nvim/option_defs.h

```diff
@@ -59,6 +59,9 @@
 #define EOL_DOS         1       /* CR NL */
 #define EOL_MAC         2       /* CR */
 
+#define DFLT_COLS       80      // default value for 'columns'
+#define DFLT_LINES      24      // default value for 'lines'
```

Comment:
Forgot about those! I'm away from my laptop, so If you think this is mergeable I would kindly ask you to make that change. Or we can wait ;)